CONTRIBUTING.md

```diff
@@ -24,15 +23,11 @@ Our [contributor guide for individuals](contributor-guides/for-individuals.md) e
 
 We’re especially interested in learning what you need to implement the [Standard for Public Code](https://standard.publiccode.net/) in your organization. Please [raise an issue](https://standard.publiccode.net/CONTRIBUTING.html) or email us at <info@publiccode.net>.
 
-We're currently hiring codebase stewards - [see our careers and open positions](https://publiccode.net/careers).
-
-[Join our mailing list] to keep up with our news.
+[Join our mailing list](https://docs.google.com/forms/d/e/1FAIpQLSdOArHWz6leVrOdOLcVlm9nbgKSrn5gxAgCIytZsAWgqnkbWQ/viewform) to join our community and keep up with our news.
```

Comment:
"Join .. to join" is a bit repeating. Perhaps make the linked  words "Sign up" instead?